src/components/menu/menu.spec.ts

```diff
@@ -376,6 +383,38 @@ describe('MenuComponent', () => {
 
         expect(component.item1Activated).toHaveBeenCalledTimes(1);
     });
+
+    it('should not emit a click event when disabled is true.', async () => {
+        component.disabled = true;
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // open menu
+        component.trigger.openMenu();
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // get the child submenu item
+        const menuItem = document.querySelector<HTMLButtonElement>(
+            '#menu-item-disabled'
+        );
+        expect(menuItem).toBeTruthy();
+
+        const item1Element = overlayContainerElement.querySelector('#menu-item-1') as HTMLButtonElement;
+        expect(item1Element).toBeTruthy();
+
+        spyOn(component, 'item2Click');
+        spyOn(component, 'item1Activated');
+
+        // perform a click
+        item1Element.click();
+        menuItem.click();
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        expect(component.item2Click).not.toHaveBeenCalled();
```

Comment:
I don't think this test is actually clicking item 2? I presume you mean item1Click?

If you want to test clicking a submenu trigger as well as a regular menu item, that should be a separate test. I think that would be worth doing, you can check that it doesn't open the submenu.